README.md

```diff
@@ -263,6 +264,11 @@ make podman-run  # Requires .env file in project root
 export QUAY_CRED='<base64 encoded pull secret>'
 export BUGZOOKA_IMAGE='<image tag>'
 export BUGZOOKA_NAMESPACE='<your namespace>'
+# optional Jira fields
```

Comment:
These fields are specific to JIRA mcp. They should only be documented as a part of the integration. Please kindly remove them